test/images/sample-apiserver/Makefile

```diff
@@ -24,7 +24,7 @@ export
 # Get without building to populate module cache
 # Then, get with OS/ARCH-specific env to build
 bin:
-	docker run --rm -i -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.16.4-1 \
+	docker run --rm -i -v "${TARGET}:${TARGET}:Z" k8s.gcr.io/build-image/kube-cross:v1.16.4-2 \
```

Comment:
I think if we make this a `$(cat ...)` command for `VERSION` then we would not have to request approval for every kube-cross build.

Reply:
+1

Reply:
+1 from me @saschagrunert